app/controllers/api/v1/credentials_controller.rb

```diff
@@ -3,7 +3,16 @@ class CredentialsController < ApiController
     respond_to :json
 
     def show
-      render json: CredentialsSerializer.new(current_resource_owner).call
+      render json: credentials_serializer.call
```

Comment:
Funny thing I started doing just that but decided it a larger piece of work and was outside the scope of this PR
Will do separately